docs/deep-linking.md

```diff
@@ -40,12 +40,50 @@ const SimpleApp = createAppContainer(createStackNavigator({...}));
 // on Android, the URI prefix typically contains a host in addition to scheme
 // on Android, note the required / (slash) at the end of the host property
 const prefix = Platform.OS == 'android' ? 'mychat://mychat/' : 'mychat://';
+// in Expo projects you can do the following to get the URI prefix
+const prefix = Expo.Linking.makeUrl('/');
 
 const MainApp = () => <SimpleApp uriPrefix={prefix} />;
 ```
 
 ## Set up with Expo projects
 
+you need to specify a scheme for your app. You can register for a scheme in your `app.json` by adding a string under the scheme key:
```

Comment:
should capitalize the first word in the paragraph, so "You need" rather than "you need"

Reply:
🤦🏾‍♂️
sorry haha